File: backend/Models/Review.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend.Models
{
    public class Review
    {
        public int Id { get; set; }
        public int Grade { get; set; }
        public string Comment { get; set; }
        public DateTime DateReviewed { get; set; }
        public int SalonId { get; set; }
        public Salon Salon { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }

    }
}

File: frontend/src/app/models/review.ts
import { Salon } from './salon';
import { User } from './user';

export interface Review {
  id: number;
  grade: number;
  comment: string;
  dateReviewed: Date;
  salonId: number;
  salon: Salon;
  userId: number;
  user: User;
}
